ui/src/app/cohort-search/list-search-v2/list-search-v2.component.tsx

```diff
@@ -378,78 +361,56 @@ export const ListSearchV2 = fp.flow(withCdrVersions(), withCurrentWorkspace())(
-            <thead className='p-datatable-thead'>
-              <tr style={{height: '2rem'}}>
-                <th style={{...styles.columnHeader, borderLeft: 0}}>Name</th>
-                <th style={{...styles.columnHeader, width: '20%'}}>Code</th>
-                <th style={{...styles.columnHeader, width: '10%'}}>Vocab</th>
-                <th style={{...styles.columnHeader, width: '8%'}}>Count</th>
-                <th style={{...styles.columnHeader, textAlign: 'center', width: '12%'}}>View Hierarchy</th>
-              </tr>
-            </thead>
-          </table>
-          <div style={{height: '15rem', overflowY: 'auto'}}>
-            <table className='p-datatable' style={{...styles.table, ...styles.tableBody}}>
-              <tbody className='p-datatable-tbody'>
-              {displayData.map((row, r) => {
-                const open = ingredients[row.id] && ingredients[row.id].open;
-                const err = ingredients[row.id] && ingredients[row.id].error;
-                return <React.Fragment key={r}>
-                  {this.renderRow(row, false, r)}
-                  {open && !err && ingredients[row.id].items.map((item, i) => {
-                    return <React.Fragment key={i}>{this.renderRow(item, true, `${r}.${i}`)}</React.Fragment>;
-                  })}
-                  {open && err && <tr>
-                    <td colSpan={5}>
-                      <div style={{...styles.error, marginTop: 0}}>
-                        <ClrIcon style={{margin: '0 0.5rem 0 0.25rem'}} className='is-solid' shape='exclamation-triangle' size='22'/>
-                        Sorry, the request cannot be completed. Please try again or contact Support in the left hand navigation.
-                      </div>
-                    </td>
-                  </tr>}
-                </React.Fragment>;
-              })}
-            </tbody>
+          {!!displayData.length && <React.Fragment>
+            <table className='p-datatable' style={styles.table}>
+              <thead className='p-datatable-thead'>
+                <tr style={{height: '2rem'}}>
+                  <th style={{...styles.columnHeader, borderLeft: 0}}>Name</th>
+                  <th style={{...styles.columnHeader, width: '20%'}}>Code</th>
+                  <th style={{...styles.columnHeader, width: '10%'}}>Vocab</th>
+                  <th style={{...styles.columnHeader, width: '8%'}}>Count</th>
+                  <th style={{...styles.columnHeader, textAlign: 'center', width: '12%'}}>View Hierarchy</th>
+                </tr>
+              </thead>
             </table>
-          </div>
-        </React.Fragment>}
+            <div style={{height: '15rem', overflowY: 'auto'}}>
+              <table className='p-datatable' style={{...styles.table, ...styles.tableBody}}>
+                <tbody className='p-datatable-tbody'>
+                {displayData.map((row, r) => {
```

Comment:
nit: here as well can we use index instead of r

Reply:
Done